packages/office-ui-fabric-react/src/components/ChoiceGroup/ChoiceGroup.Props.ts

```diff
@@ -63,6 +63,11 @@ export interface IChoiceGroupOption {
   onRenderField?: IRenderFunction<IChoiceGroupOption>;
 
   /**
+   * Optional override of option render
+   */
+  onRenderLabel?: (IChoiceGroupOption) => JSX.Element;
```

Comment:
Hmm hadn't tried that. Thought the problem was nesting the on renders. Worth a try.